AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -446,6 +493,7 @@ - (BOOL)shouldBePlaying
 
 - (void)dealloc
 {
+  _timeObserver = nil;
```

Comment:
Should remove the observer from the player first right? 

Something like `[_player removeTimeObserver:_imeObserver];`